textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/regexmatch/RegexTestConstantsCorp.java

```diff
@@ -37,7 +37,7 @@
     	IField [] fields1 = {new StringField("Facebook"), new StringField("404 Not Found"), new StringField("66.220.144.0")};
     	IField [] fields2 = {new StringField("Weibo"), new StringField("http://weibo.com"), new StringField("180.149.134.141")};
     	IField [] fields3 = {new StringField("Microsoft"), new StringField("https://www.microsoft.com/en-us/"), new StringField("131.107.0.89")};
-    	IField [] fields4 = {new StringField("Google"), new StringField("www.google.com"), new StringField("8.8.8.8.8.8")};
+    	IField [] fields4 = {new StringField("Google"), new StringField("websit: www.google.com"), new StringField("8.8.8.8.8.8")};
```

Comment:
@sandeepreddy602 : can you confirm RegexTestConstantsCorp.java and RegexTestConstantStaff are using the right folder location?  If not, where to put them?